sdk/cognitiveservices/cognitiveservices-customvision-prediction/README.md

```diff
@@ -17,84 +17,85 @@ npm install @azure/cognitiveservices-customvision-prediction
 
 #### nodejs - Authentication, client creation and classifyImageUrl  as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-azure-js
```

Comment:
The sample doesn't use `@azure/ms-rest-azure-js`. This api doesn't need credentials?

Reply:
Same for customvision-training

Reply:
You are r8. Removed it